components/object_store/src/obkv/meta.rs

```diff
@@ -375,4 +399,16 @@ mod test {
             version: String::from("123456fsdalfkassa;l;kjfaklasadffsd"),
         }
     }
+
+    fn build_test_meta1() -> ObkvObjectMeta {
```

Comment:
If name it in this style, how about renaming `build_test_meta` to `build_test_meta0` at L382?

Reply:
ok